simapp/app.go

```diff
@@ -372,6 +374,7 @@ func NewSimApp(
 	app.SetAnteHandler(
 		ante.NewAnteHandler(
 			app.AccountKeeper, app.BankKeeper, *app.IBCKeeper, ante.DefaultSigVerificationGasConsumer,
+			amino.LegacyAminoJSONHandler{},
```

Comment:
are we updating this for 0.39 to the new JSON signing?

Reply:
Once, proto binary signing is ready, we will enable it here as well. There is no new JSON signing. We will allow amino JSON signing + proto binary signing (#6111).